client/lib/seo/index.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { includes } from 'lodash';
+
+const conflictingSeoPlugins = [
+	'Yoast SEO',
+	'Yoast SEO Premium',
+	'All In One SEO Pack',
+	'All in One SEO Pack Pro',
+];
```

Comment:
Since this was here before I assume this isn't translated, right? Eg if I have my language set as ES for example.

Reply:
👍  cool the check still works in other languages:
![screen shot 2018-01-12 at 1 59 34 pm](https://user-images.githubusercontent.com/1270189/34897021-e25f7e90-f7a0-11e7-80e1-3904ea70c41c.png)